armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -617,6 +618,10 @@ def interactEOC(self, cycle=None):
         """
         self.clearRepresentativeBlocks()
 
+    def interactEveryNode(self, cycle=None):
+        if self.cs[CONF_FORCE_LATTICE_PHYSICS]:
```

Comment:
we defiantly need this on every node so we can truncate the 1e-15 isotopes from the mc2 file. if its not in every node than the build up with depletion will cause no matching xs for those isotopes since they wont be there at BOC. 

Its optional to have it at interactCoupled, but when you do it with interactCoupled it does get you a temp update from the TH solve which this call wouldn't have.